python/tank/util/shotgun.py

```diff
@@ -236,12 +240,23 @@ def download_url(sg, url, location):
     
     # download the given url
     try:
+        request = urllib2.Request(url)
+        if sg.config.server in url:
+            request.add_header("user-agent", "; ".join(sg._user_agents))
```

Comment:
accessing an internal method in the shotgun API. Can we do this in some other way please?